tests/unit/test_handlers.py

```diff
@@ -291,10 +295,9 @@ def test_run_instances_userdata_blob(self):
         # user data.
         user_data = b'\xc7\xa9This is a test'
         b64_user_data = base64.b64encode(user_data).decode('utf-8')
-        event = 'before-parameter-build.ec2.RunInstances'
         params = dict(ImageId='img-12345678',
                       MinCount=1, MaxCount=5, UserData=user_data)
-        self.session.emit(event, params=params)
+        handlers.base64_encode_user_data(params=params)
```

Comment:
By emitting that event, it will call every handler and that fake emission of the event does not include an operation model which is required by the alias handler causing a runtime error. By removing the emission of an event, I feel like that it has become more of a unit test as well as it is decoupled from all of the other handlers that may be registered to this event. Thoughts?